README.md

```diff
@@ -1 +1,55 @@
-# Review-Admissions-Trial-Project
\ No newline at end of file
+# Review-Admissions-Trial-Project
```

Comment:
Please don't leave unused comment in your code-base. Unused code as comment makes your code not clean.